packages/console/src/pages/Connectors/index.tsx

```diff
@@ -1,68 +1,125 @@
+        ],
+        [
+          data.find((connector) => connector.metadata.type === ConnectorType.SMS),
+          t('connectors.type.sms'),
+        ],
+      ];
+    }, [data, isSocial, t]);
+
+  const socialConnectors = useMemo(() => {
+    if (!isSocial) {
+      return;
+    }
+
+    return data?.filter((connector) => connector.metadata.type === ConnectorType.Social);
+  }, [data, isSocial]);
+
   return (
     <Card>
       <div className={styles.headline}>
         <CardTitle title="connectors.title" subtitle="connectors.subtitle" />
-        <Button disabled title="admin_console.connectors.create" />
+        {isSocial && <Button disabled title="admin_console.connectors.create" />}
       </div>
+      <TabNav>
+        <TabNavLink href="/connectors">{t('connectors.tab_email_sms')}</TabNavLink>
+        <TabNavLink href="/connectors/social">{t('connectors.tab_social')}</TabNavLink>
+      </TabNav>
       <table className={styles.table}>
         <thead>
           <tr>
             <td className={styles.connectorName}>{t('connectors.connector_name')}</td>
+            <td>{t('connectors.connector_type')}</td>
             <td>{t('connectors.connector_status')}</td>
           </tr>
         </thead>
         <tbody>
           {error && (
             <tr>
-              <td colSpan={2}>error occurred: {error.metadata.code}</td>
+              <td colSpan={3}>error occurred: {error.metadata.code}</td>
             </tr>
           )}
           {isLoading && (
             <tr>
-              <td colSpan={2}>loading</td>
+              <td colSpan={3}>loading</td>
+            </tr>
+          )}
+          {socialConnectors?.length === 0 && (
+            <tr>
+              <td colSpan={3}>
+                <div className={styles.empty}>
+                  <div>
+                    <img src={placeholder} />
+                  </div>
+                  <div className={styles.emptyLine}>{t('connectors.type.social')}</div>
+                  <div className={styles.emptyLine}>{t('connectors.social_connector_eg')}</div>
+                  <Button disabled title="admin_console.connectors.create" />
+                </div>
+              </td>
             </tr>
           )}
-          {data?.map(({ id, metadata: { name, logo }, enabled }) => (
-            <tr key={id}>
+          {emailAndSmsConnectors?.map(([connector, type]) => (
+            <tr key={type}>
+              <td>
+                <ConnectorName connector={connector} titlePlaceholder={type} />
+              </td>
+              <td>{type}</td>
+              <td>
+                {connector ? (
+                  <Status status="operational">{t('connectors.connector_status_enabled')}</Status>
+                ) : (
+                  <Button title="admin_console.connectors.set_up" />
+                )}
+              </td>
+            </tr>
+          ))}
+          {socialConnectors?.map((connector) => (
+            <tr key={connector.id}>
```

Comment:
Ditto.
Make this component logic become a local component like `ConnectorName`.